app-admin/lnav/lnav-0.8.4.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A curses-based tool for viewing and analyzing log files"
+HOMEPAGE="https://lnav.org"
+SRC_URI="https://github.com/tstack/lnav/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="unicode"
+
+RDEPEND="
+	dev-db/sqlite:3
+	dev-libs/libpcre[cxx]
+	>=net-misc/curl-7.23.0
+	sys-libs/ncurses:0=[unicode?]
+	sys-libs/readline:0=
+	sys-libs/zlib"
+DEPEND="${RDEPEND}
+	app-arch/bzip2"
```

Comment:
no, this is not needed at all, we do not include stuff from the @system set explicitly without good enough reason.

Reply:
Okay, bzip2 removed.